src/lib.rs

```diff
@@ -163,6 +163,9 @@ pub fn get_region_from_path<P: AsRef<Utf8Path>>(path: P) -> Option<Region> {
     } else {
         None
     }
+extern "C" {
+    #[link_name = "_ZN2nn2fs17MountCacheStorageEPKc"]
+    fn mount_cache_storage(string: *const u8);
```

Comment:
Move this to [nnsdk-rs](https://github.com/ultimate-research/nnsdk-rs), because this could be used by other people in other games.

The [fs](https://github.com/ultimate-research/nnsdk-rs/blob/master/src/lib.rs#L2851) module already exists, so feel free to add it there and make a Rust wrapper for it.

If you don't know what it is, it is a Rust method that accepts Rust types and converts things safely to be provided to unsafe function.
In this case, it'd be a method that takes a &str and provided it to skyline::cstr while appending the '\0'.

Here is an [example](https://github.com/Raytwo/nn-fuse/blob/845aab8e8bc01c0a45b5c431b73b9c2df0e796e6/src/lib.rs#L53-L58) of a wrapper having to append a '\0' in nn-fuse which you can pretty much duplicate aside from the ``= 0``part.

Also small nitpick but when moving the extern function to nnsdk-rs, make sure to reproduce the spelling of nnsdk so the distinction is clear (MountCacheStorage), as you're using Rust conventions for an external function which can be misleading.
That way it becomes nn::fs::MountCacheStorage when adding the module, which is the official spelling.